app/main/views/buyers.py

```diff
@@ -414,9 +419,10 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
 
     brief_responses = data_api_client.find_brief_responses(brief_id)['briefResponses']
 
-    brief_responses_require_evidence = (
-        datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
-        <= datetime.strptime(brief['publishedAt'][0:10], "%Y-%m-%d")
+    brief_responses_required_evidence = (
+        None
+        if not brief_responses else
+        not is_legacy_brief_response(brief_responses[0], brief=brief)
```

Comment:
Cool 👍I think it's a better explanation of what's happening than the alternative I just thought of... 

... hardcoding a check for `brief['publishedAt'][0:10] > '2017-02-08'` - I mean, we wouldn't need separate checks for preview/staging as that data has long been wiped, but it feels a bit icky.

Reply:
Yeah, mine still isn't an ideal way, in that `None` is falsey, but meh...